qgreenland/config/layers.yml

```diff
@@ -263,6 +263,27 @@
   ogr2ogr_kwargs:
     OGR_ENABLE_PARTIAL_REPROJECTION: True
 
+- id: exclusive_economic_zone
+  title: 'Exclusive Economic Zone'
+  description: 'Exclusive Economic Zone of Greenland based on the 1973 Agreement'
+  data_source: 1973_agreement.only
+  visible: False
+  group_path: 'Miscellaneous'
```

Comment:
I feel like we already have an empty group somewhere for this... `Environmental management/Exclusive economic zones`!

Reply:
There's only one official "Exclusive Economic Zone" right? To everyone, that means this one thing? In that case, let's get rid of the group, and replace it with this layer in `Environmental management` group. If not, maybe we should further qualify the `id` and `title` of this one to `1973_agreement_exclusive_economic_zone`, and insert it into that group.

Reply:
Hmm, yeah. We'll need more info on this. Maybe the empty group was wrong, as this layer is in 'Misc' in our layer planning spreadsheet.